src-java/testing/test-library/src/main/java/org/openkilda/testing/config/DefaultServiceConfig.java

```diff
@@ -131,6 +132,29 @@ public List<Floodlight> floodlights(@Value("#{'${floodlight.openflows}'.split(',
         return floodlights;
     }
 
+    @Bean
+    @Scope(SCOPE_PROTOTYPE)
+    public List<Floodlight> commonFloodlights(@Value("#{'${floodlight.openflows}'.split(',')}") List<String> openflows,
+                                        @Value("#{'${floodlight.endpoints}'.split(',')}") List<String> endpoints,
+                                        @Value("#{'${floodlight.containers}'.split(',')}") List<String> containers,
+                                        @Value("#{'${floodlight.regions}'.split(',')}") List<String> regions,
+                                        @Value("#{'${floodlight.modes}'.split(',')}") List<String> modes,
+                                        @Autowired TopologyDefinition topo) {
+        if (openflows.size() != endpoints.size() || openflows.size() != containers.size()
+                || openflows.size() != regions.size() || openflows.size() != modes.size()) {
+            throw new IllegalArgumentException("Floodlight test properties are illegal. Sizes of floodlight.openflows,"
+                    + " floodlight.endpoints, floodlight.containers, floodlight.regions, floodlight.modes should be "
+                    + "equal");
+        }
+        List<Floodlight> floodlights = new ArrayList<>();
```

Comment:
Probably this would be shorter with streams api.

Reply:
As we use the same index for the rest of the props (openflows.get(i), containers.get(i), regions.get(i), modes.get(i)) to set up each floodlight, it seems like using streams is not applicable here.

Reply:
Hm. AI says, that it can be written as 
```
return IntStream.range(0, regions.size())
    .mapToObj(i -> new Floodlight(
        openflows.get(i),
        containers.get(i),
        regions.get(i),
        new FloodlightServiceImpl(endpoints.get(i)),
        EnumUtils.getEnumIgnoreCase(FloodlightConnectMode.class, modes.get(i))
    ))
    .collect(Collectors.toList());
 ```
 , but it it doesn't look more readable. 
 I'd still move each `Floodlight` object constructor parameter to the new line, but it's not mandatory in our coding convention.